backend/LeetWars.Core/LeetWars.Core.BLL/Services/UserService.cs

```diff
@@ -105,6 +118,30 @@ public async Task<List<UserSolutionsGroupedBySkillLevelDto>> GetUserChallengesIn
             .ToListAsync();
 
         return challenges;
+    }
+
+    public async Task<UserFullDto> UpdateUserAsync(EditUserDto userDto)

```

Comment:
probably this should be not just the UpdateUserAsync method as we will have update functionality soon

It would be better to give a more specific name like UpdateUserScore or something like that so we can reuse it during challenge applying